### PR TITLE
Feature/1732 generate data extra

### LINKED ISSUE #1732: Add selectielijstklasse and archief datum to test data generation.
### Thema / Theme

Zaken API

### Omschrijving / Description

The (generate_data?) management command creates test data but seems to result in a set of zaken that lacks a (valid) selectielijstklasse and an archiefdatum. 

### Toegevoegde waarde / Added value

This allows more realistic test data to be generated benefitting the development of OAB.

### Aanvullende opmerkingen / Additional context

_No response_

### PR DESCRIPTION
**Pull request by annashamray**
Fixes #1732

### CODE REVIEW

File: src/openzaak/management/commands/generate_data.py [hunk 1]
```diff
@@ -64,6 +65,19 @@
     ZaakInformatieObjectFactory,
     ZaakObjectFactory,
 )
+from openzaak.selectielijst.models import ReferentieLijstConfig
+
+
+@vcr.use_cassette("src/openzaak/tests/cassettes/selectielijst_resultaten.yaml")
```
Comment (bart-maykin) [inline src/openzaak/management/commands/generate_data.py:71]: Any reason why we are using vcr to create a cassette? I don't see us using this for anything atm.

**Review by bart-maykin — COMMENTED** [review]
Seems good although I don't see any tests.
Did you forget to push those by any chance?

File: src/openzaak/management/commands/generate_data.py (hunk 1 above)
Comment (annashamray) [reply src/openzaak/management/commands/generate_data.py:71]: I've read vcr documentation one more time and noticed that they recommend to use it with tests. There is no example o use vcr outside of tests. So after some consideration I decided not to use it. So let's just call Selectielijst API resource for real in this command

**Comment by annashamray** [comment]
@bart-maykin I've updated a test and added assertions for Zaak attributes

**Review by bart-maykin — APPROVED** [review]

**Pull request merged**